# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/canvas/test/webgl-conf/generated-mochitest.ini
# Commit: e1b0906509ef
# Full Hash: e1b0906509efa6433978b9f980ec639b7eafbc6d
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-01-09 03:40:18
# Regressor Bug: 1477756
# File Overlap Count: 1
# Description:
#   Bug 1477756 - Fix non-webgl CI tests. r=handyman
#   
#   * Revert some partial webgl+oop+vr code.
#   * More missing FuncScope.
#   * Fix compile errors.
# ==============================================================================

diff -r 7e2a2b1b416f -r e1b0906509ef dom/canvas/test/webgl-conf/generated-mochitest.ini
--- a/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 08 22:19:23 2020 +0000
+++ b/dom/canvas/test/webgl-conf/generated-mochitest.ini	Wed Jan 08 22:19:26 2020 +0000
@@ -12100,7 +12100,7 @@
 subsuite = webgl1-core
 [generated/test_conformance__rendering__blending.html]
 subsuite = webgl1-core
-fail-if = (os == 'android') || (os == 'linux')
+fail-if = (os == 'android')
 [generated/test_conformance__rendering__canvas-alpha-bug.html]
 subsuite = webgl1-core
 [generated/test_conformance__rendering__clear-after-copyTexImage2D.html]